src/snowflake/snowpark/session.py

```diff
@@ -717,11 +734,23 @@ def _resolve_packages(
                         f"and add it via session.add_import()."
                     )
                 elif not use_local_version:
+                    # check if user as requested for a specific version of package 'pkg==version'
+                    if package_req.specs:
+                        package_version_req = package_req.specs[0][1]
+                        if package_version_req not in valid_packages[package_name]:
+                            raise ValueError(
+                                f"Cannot add package {package_name} {package_version_req} "
```

Comment:
I'm wondering if we should have a helper function for this error message as this looks very similar to the one above and the only difference is `{package_version_req}`. This way if we want to modify the message, we run a lower risk of forgetting one.

Reply:
Do we have a specific file where we put these helper functions or should I create a function locally?